netutils/interface.py

```diff
@@ -237,7 +251,7 @@ def abbreviated_interface_name(interface, addl_name_map=None, addl_reverse_map=N
     return interface
 
 
-@total_ordering
+@total_ordering  # type: ignore
```

Comment:
We could add this issue to this comment: https://github.com/python/mypy/issues/5374